test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -285,7 +285,7 @@ describe('ReactRefreshLogBox app', () => {
     } else {
       await expect(browser).toDisplayRedbox(`
        {
-         "description": "Error:   x Unexpected token. Did you mean \`{'}'}\` or \`&rbrace;\`?",
+         "description": "  x Unexpected token. Did you mean \`{'}'}\` or \`&rbrace;\`?",
```

Comment:
The spacing and `x` was already misplaced. We take on build errors another time though. This is a step in the right direciton.